### DISCUSSION TITLE
Helpful information for users on account creation

### DISCUSSION THREAD

**Discussion by alexrandaccio** [RadicalxChange/rxc-voice#89] [General]
What information should we provide to users on the create account page? @patcon @DecimalTurn 

![Screen Shot 2022-01-19 at 3 41 49 PM](https://user-images.githubusercontent.com/48193894/150210875-6b2ec392-c98f-444e-85a4-aa2053ba2951.png)

![Screen Shot 2022-01-19 at 3 46 43 PM](https://user-images.githubusercontent.com/48193894/150211156-d69495d0-a3f2-45d0-818c-163509f3e8f2.png)

**Comment by patcon** [comment]
> What information should we provide to users on the create account page?

The consequences and perks of choosing one identity verifier over another (GitHub or Twitter).
- be clear that my identity on that platform will be exposed to anyone I interact with on the platform, and
- in telling me how it's exposed, help me decide which identity is most appropriate (e.g., if I'm expecting to use RxC Voice for non-tech discussions, I don't want people clicking through to my GitHub profile).

Something like:

> Verify your identity, and publicly associate that profile with your actions on RxC Voice.
> <details>
> <summary><strong>(?)</strong></summary>
> We will not access any information on your third-party account other than your username.
> </details>

**Comment by DecimalTurn** [comment]
@alexrandaccio 
On a related note, is there a good reason to ask for the verification method as soon as the person creates their profile? Could we keep that step for the next screen after the first form is submitted?

Also related: https://github.com/RadicalxChange/rxc-voice/issues/87

**Reply by alexrandaccio** [reply]
@leoneric also suggested this from the very beginning and I keep forgetting to do it. I'll open an issue

EDIT: see issue #92
